geoshape/track.json

```diff
@@ -0,0 +1,73 @@
+{% import "rally.helpers" as rally with context %}
+
+{
+  "version": 2,
+  "description": "Shapes from PlanetOSM",
+  "indices": [
+    {
+      "name": "osmlinestrings",
+      "body": "index.json",
+      "types": ["_doc"]
+    },
+    {
+      "name": "osmmultilinestrings",
+      "body": "index.json",
+      "types": ["_doc"]
+    },
+    {
+      "name": "osmpolygons",
+      "body": "index.json",
+      "types": ["_doc"]
+    }
+  ],
+  "corpora": [
+    {
+      "name": "linestrings",
+      "base-url": "http://benchmarks.elasticsearch.org.s3.amazonaws.com/corpora/geoshape",
+      "target-index": "osmlinestrings",
+      "target-type": "_doc",
```

Comment:
Support for types has been removed on master, so this works against 6.x but not against master.